apps/crn-frontend/src/shared-research/__tests__/ResearchOutput.test.tsx

```diff
@@ -101,23 +117,64 @@ describe('a grant document research output', () => {
       ],
     });
 
-    const { getByText } = await renderComponent();
+    const { getByText } = await renderComponent(researchOutputRoute.$);
     expect(getByText('Team Sulzer, D')).toHaveAttribute(
       'href',
       expect.stringMatching(/0d074988-60c3-41e4-9f3a-e40cc65e5f4a/),
     );
   });
+
+  it('renders the edit page when you have permissions', async () => {
+    mockGetResearchOutput.mockResolvedValue({
+      ...createResearchOutputResponse(),
+      documentType: 'Bioinformatics',
+      teams: [
+        {
+          id: 't0',
```

Comment:
can we have this ID hardcoded into the user? You shouldn't rely on mock data to make a test work